components/typography/typography.scss

```diff
@@ -111,11 +111,14 @@
 	color: colors.$d2l-color-tungsten;
 	font-size: 0.7rem;
 	font-weight: 400;
-	line-height: 1rem;
+	line-height: 0.9rem;
 	margin: auto;
 	@media (max-width: 615px) {
 		font-size: 0.6rem;
-		line-height: 0.9rem;
+		line-height: 0.9rem; /* TODO: what to do with this? */
+	}
+	.d2l-typography.d2l-no-input-style-tweaks & {
```

Comment:
The LMS is going to output this CSS class when the flag is OFF. That lets us keep the main rule clean and just delete this rule when the flag goes away.